src/components/PopupWithForm.js

```diff
@@ -35,11 +36,27 @@ export default class PopupWithForm extends Popup {
     this._form.addEventListener("submit", (event) => {
       event.preventDefault();
 
+    
+
       this._handleFormSubmit(this._getInputValues());
     });
   }
 
   getForm() {
     return this._form;
   }
+
+  handleAvatarModal() {
+    this._handleAvatarModal;
+    this._form.addEventListener.open();
```

Comment:
Please, delete `handleAvatarModal`  from `PopupWithForm` because you don't use it and you can 't use it

There should **not** be any code in universal functions (classes) that doesn’t suit every popup.